### PR TITLE
1.1.2 alpha.1

### PR DESCRIPTION
**Pull request by 6l20**
## PR Description

- Agents array
- existingPVC
- QKM 21.9.3

## Documentation

- [ ] I thought about documentation and added the `documentation` label to this PR if updates are required.

### CODE REVIEW

File: templates/deployment.yaml [hunk 1]
```diff
@@ -114,18 +210,28 @@ spec:
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
       volumes:
-      {{- if .Values.storage.nfs.enabled }}
         - name: vault-token
+      {{- if empty .Values.storage.existingPvc }}
```
Comment (ggarri) [inline templates/deployment.yaml:214]: I believe an easier logic would be:
- if exitingPvc is defined, then use it 
- elseif nfs is enabled, then use it
- else, do nothing

Comment (6l20) [reply templates/deployment.yaml:216]: Ok, Well you mean the inverted case ?
- if existingPvc claimName = existingPvc
- elif nfs claimName = nfsPvc
- else emptyDir

Comment (ggarri) [reply templates/deployment.yaml:216]: Fine like that

File: templates/init-migrate-hook.yaml [hunk 2]
```diff
@@ -0,0 +1,110 @@
+{{- if .Values.migrate.enabled }}
```
Comment (ggarri) [inline templates/init-migrate-hook.yaml:1]: Migration MUST already run

**Pull request merged**